cirq/google/engine/engine_sampler.py

```diff
@@ -0,0 +1,59 @@
+from typing import Union, List, TYPE_CHECKING
+
+from cirq import work
+from cirq.google.engine import engine
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import
+    import cirq
+
+
+class QuantumEngineSampler(work.Sampler):
+    """A sampler that samples from real hardware on the quantum engine."""
+
+    def __init__(self,
+                 *,
+                 google_cloud_project_id: str,
+                 processor_id: Union[str, List[str]],
+                 gate_set: 'cirq.google.SerializableGateSet'):
+        """
+        Args:
+            google_cloud_project_id: The identifier of the google cloud project
```

Comment:
Should this just be 'project_id' to match the existing identifiers in the Engine?

Reply:
link to google cloud documentation of what this id is

Reply:
Obsoleted by taking engine as a constructor argument.